streaming/README.md

```diff
@@ -511,4 +511,88 @@ metrics, following the same pattern of the receiving side.
 
 For a practical example see [Monitoring ephemeral nodes](#monitoring-ephemeral-nodes).
 
+## Troubleshooting streaming connections
+
+Sometimes when we are configuring master and slave nodes it is possible to have some errors in the log files, this section
+describes these errors in the communication.
+
+### Connection between slave and master is slow
+
+When we have a slow connection between master and `slave`, Netdata can raise different errors related to this problem,
```

Comment:
If you are going to use backticks for slave, you might as well use them for master too.

Reply:
In the beginning I was writing backticks to call attention of the people for both master and slave, but when I began to review I saw that this was not a pattern, after this I began to remove, but I forgot this one. Thank you!